extension/js/common/composer/composer-contacts.ts

```diff
@@ -38,13 +38,51 @@ export class ComposerContacts extends ComposerComponent {
   }
 
   initActions(): void {
-    this.composer.S.cached('input_to').on('keyup', Ui.event.prevent('veryslowspree', () => this.searchContacts()));
+    let preventSearchContacts = false
+    this.composer.S.cached('input_to').on('keyup', Ui.event.prevent('veryslowspree', () => {
+      if (!preventSearchContacts) {
+        this.searchContacts()
```

Comment:
@tomholub what would be the right way to resolve the `Promises must be handled appropriately` error from the code quality check?

Reply:
Most likely you're missing an `await` somewhere. Possibly in front of `this.searchContacts()`, which means you'd also put `async` in front of `() => {`.

If you don't await a promise, any promise rejections will be ignored / errors silently swallowed, which is what it's trying to keep you from doing.

Reply:
Done, thanks! :+1: